### PR TITLE
data:MoNihal10

### PR DESCRIPTION
**Pull request by MoNihal10**

### CODE REVIEW

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
Hey @MoNihal10 Fix your file name, it should be `data/MoNihal10.json`

File: data/MoNihal.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name" : "Mohammed Nihal" ,
+  "displayStatsPublic" :true ,
+  "type" : "personal" ,
+  "bio" : " cs undergrad!!! " ,
+  "avatar" : "https://github.com/MoNihal10.png",
+  "links" :[
+  {
+    "name" : "Follow me on Twitter" ,
+    "url" : "https://twitter.com/n_ihal_10"
```
Comment (Aadarsh805) [inline data/MoNihal.json:10]: and a missing comma right here at the end of line 10

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/MoNihal.json (hunk 1 above)
Comment (MoNihal10) [reply data/MoNihal.json]: I have made the changes.

Comment (MoNihal10) [reply data/MoNihal.json]: Now, do I have to close the pull request and reopen it??

Comment (Aadarsh805) [reply data/MoNihal.json]: Nah it's already open, you don't need to do anything

Comment (MoNihal10) [reply data/MoNihal.json]: How long does it usually take to merge a pull request and get the website to start running in my custom url??

Comment (Aadarsh805) [reply data/MoNihal.json]: > How long does it usually take to merge a pull request and get the website to start running in my custom url??

It depends, sometimes maintainers are busy with other things right
But it doesn't take more than a day, so you'll have it by tomorrow for sure

Comment (MoNihal10) [reply data/MoNihal.json]: Alright thanks!!

File: data/MoNihal10.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "icon" : "FaTwitter"
+  },
+  {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/n_ihal_10",
+      "icon": "FaInstagram"
+    },
+   {
+      "name": "LinkedIn",
+      "url": "www.linkedin.com/in/mohammed-nihal-b80b6b227",
```
Comment (EmmaDawsonDev) [inline data/MoNihal10.json:20]:
```suggestion
      "url": "https://www.linkedin.com/in/mohammed-nihal-b80b6b227",
```

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Just a small change to make to the LinkedIn url, then this should be good to go.

File: data/MoNihal10.json (hunk 2 above)
Comment (MoNihal10) [reply data/MoNihal10.json]: the changes have been done.

**Review by EmmaDawsonDev — APPROVED** [review]
Looks good 👍

File: data/MoNihal10.json (hunk 2 above)
Comment (MoNihal10) [reply data/MoNihal10.json]: when is the custom web URL for my account going to work???

Comment (Aadarsh805) [reply data/MoNihal10.json]: > when is the custom web URL for my account going to work???

As soon as this is merged, a few minutes after that

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**